LFS171x/docs/introduction-to-hyperledger-indy.md

```diff
@@ -0,0 +1,689 @@
+
+* GD2md-html version 1.0β11
+* Mon Sep 10 2018 16:05:19 GMT-0700 (PDT)
+* Source doc: https://docs.google.com/a/cloudcompass.ca/open?id=1nn7T2j24-3QsCVW4cAhUMWcHDXu1VEfn0T5cm_hauBw
+----->
+
+
+
+# Introduction and Learning Objectives
+
+
+## Introduction to Hyperledger Indy
+
+Welcome to the Hyperledger Indy section of Blockchain for Business!  In this chapter, you will learn about Hyperledger Indy - a blockchain-based system that is quite different from the other Hyperledger projects we've looked at so far. Where the other projects are general purpose blockchain systems (they can be used in many situations), Hyperledger Indy is used for just one purpose - and it's a big one! Indy is all about Identity on the Internet. It's about being able to prove to others who you are and you being certain who they are.
+
+![HyperledgerIndyLogo](../images/introduction-to-hyperledger-indy/HLindy.png "Hyperledger Indy Logo")
+
+What's the big deal about Identity?  Well, when the Internet was first created, all of the computers that were connecting to one another were "trusted." The number of systems was small and the people running those systems knew each other so they didn't need mechanisms to know who was sending what data between the systems. As the number of systems on the Internet grew (and grew and grew…), that trust quickly diminished and the first of many mechanisms were added to systems to identify who is contacting whom. Unfortunately, the problem has never really been solved. The most common (and universally hated) system, that of userIDs and passwords, is fraught with problems. The resulting lack of certainty of who is on the other keyboard has led to the loss of billions from hacks, data breaches, identity theft, scams and more. Further, that same lack of certainty has made many types of business transactions on the Internet impossible - the risk is just too high.
+
+Hyperledger Indy has been created to add an Identity layer to the Internet using a mechanism that is easy to use, enables online trust, and enhances privacy. It's a big goal that is of vital importance to everyone on the Internet. And, it's a goal that has recently become realizable with the advent of blockchain.
+
+In this chapter, we'll look in more detail at the identity problem and how Hyperledger Indy uses Decentralized Identifiers (DIDs) and Verifiable Credentials to add Identity as a core component of the Internet. We'll look at the blockchain elements of Hyperledger Indy, comparing Indy's approach to blockchain with the other systems we've looked at in this course.
+
+With that background, we'll go through a couple of Indy demonstrations - a video showing DIDs and Verifiable Credentials being used, and running a local instance of Indy that demonstrates Indy's software Agents communicating to prove who's who on the Internet. In doing that, we'll dig into to a question that is important to everyone that looks into Indy (and other Blockchains) - what goes on the ledger?

```

Comment:
"into to" -> "into"

Reply:
@tkuhrt - suggest you review the Google doc for typos, content, concepts etc. as that is still in the source of the materials.  This should be used to make sure that the markdown is properly structured and to look at the images.

Reply:
I am having trouble with my PRs as well